modules/programs/rofi-pass.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.rofi.pass;
+in {
+  meta.maintainers = [ maintainers.seylerius ];
+
+  options.programs.rofi.pass = {
+    enable = mkEnableOption "Enable rofi integration with password-store";
+    root = mkOption {
```

Comment:
root is confusing ? would rename it to `stores`.

Reply:
Good idea, fixed in d588a19.